catalog/task/update-infra-deployments/0.1/README.md

```diff
@@ -0,0 +1,23 @@
+# update-infra-deployments task
+
+Clones redhat-appstudio/infra-deployments repository, runs a script obtained from the infra-deployment-update-script key in extraData, generates pull-request for redhat-appstudio/infra-deployments repository.
+
+
+## Parameters
+|name|description|default value|required|
+|---|---|---|---|
```

Comment:
We typically use camelCase for task parameters

Reply:
applied throughout